docs/src/index.md

```diff
@@ -72,9 +72,9 @@ The default units for various quantities are:
 | :---:         |    :----:   |
 | Length        |  μm         |
 | Time          |  μs         |
-| ``\Omega``    |  2π * MHz        |
+| ``\Omega``    |  MHz        |
 | ``\phi``      |  rad        |
-| ``\Delta``    |  2π * MHz        |
```

Comment:
I think originally the 2pi factors were added to the line with both experimental & simulation units I'm wondering what has changed since then? 

cc: I think @Wang-Shengtao was in that discussion too back then when we have the 2pi factors

Reply:
Its a bit confusing, e.g. does the value of Omega/Delta need to have the two-pi multiplied by it or does Bloqade automatically multiply the factor of two-pi when you construct the hamiltonian?

Reply:
yes, I've just chatted with Phillip and John - we are deciding to change the unit to `rad/μs`.